docs/integrations/sources/github.md

```diff
@@ -17,45 +17,44 @@ This connector outputs the following full refresh streams:
 * [Issue events](https://docs.github.com/en/rest/reference/issues#list-issue-events-for-a-repository)
 * [Issue milestones](https://docs.github.com/en/rest/reference/issues#list-milestones)
+* [Issues](https://docs.github.com/en/rest/reference/issues#list-repository-issues)
+* [Project cards](https://docs.github.com/en/rest/reference/projects#list-project-cards)
+* [Project columns](https://docs.github.com/en/rest/reference/projects#list-project-columns)
 * [Projects](https://docs.github.com/en/rest/reference/projects#list-repository-projects)
+* [Pull request stats](https://docs.github.com/en/rest/reference/pulls#get-a-pull-request)
 * [Pull requests](https://docs.github.com/en/rest/reference/pulls#list-pull-requests)
 * [Releases](https://docs.github.com/en/rest/reference/repos#list-releases)
 * [Review comments](https://docs.github.com/en/rest/reference/pulls#list-review-comments-in-a-repository)
+* [Reviews](https://docs.github.com/en/rest/reference/pulls#list-reviews-for-a-pull-request)
 * [Stargazers](https://docs.github.com/en/rest/reference/activity#list-stargazers)
-* [Deployments](https://docs.github.com/en/rest/reference/deployments#list-deployments)
-* [Project cards](https://docs.github.com/en/rest/reference/projects#list-project-cards)
-* [Project columns](https://docs.github.com/en/rest/reference/projects#list-project-columns)
 
 ### Notes
 
-1. Only 3 streams from above 12 incremental streams \(`comments`, `commits` and `issues`\) are pure incremental meaning that they:
+1. Only 4 streams \(`comments`, `commits`, `issues` and `review comments`\) from above 17 incremental streams are pure incremental meaning that they:
    * read only new records;
    * output only new records.
 
-     Other 8 incremental streams are also incremental but with one difference, they:
-
+2. Other 13 incremental streams are also incremental but with one difference, they:
    * read all records;
    * output only new records.
+   Please, consider this behaviour when using those 13 incremental streams because it may affect you API call limits.
 
-     Please, consider this behaviour when using those 8 incremental streams because it may affect you API call limits.
-2. We are passing few parameters \(`since`, `sort` and `direction`\) to GitHub in order to filter records and sometimes for large streams specifying very distant `start_date` in the past may result in keep on getting error from GitHub instead of records \(respective `WARN` log message will be outputted\). In this case Specifying more recent `start_date` may help.
+3. We are passing few parameters \(`since`, `sort` and `direction`\) to GitHub in order to filter records and sometimes for large streams specifying very distant `start_date` in the past may result in keep on getting error from GitHub instead of records \(respective `WARN` log message will be outputted\). In this case Specifying more recent `start_date` may help.
 
```

Comment:
````suggestion
**The "Start Date" configuration option does not apply to the streams below, because the Github API does not include dates which can be used for filtering:**

* `assignees`
* `branches`
* `collaborators`
* `issue_labels`
* `organizations`
* `pull_request_commits`
* `pull_request_stats`
* `repositories`
* `tags`
* `teams`
* `users`
````